pkgs/tools/security/mitm6/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, fetchPypi
+, python3
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "mitm6";
+  version = "0.3.0";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-g+eFcJdgP7CQ6ntN17guJa4LdkGIb91mr/NKRPIukP8=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [
+    scapy
+    future
+    twisted
+    netifaces
+  ];
+
+  doCheck = false;
+
+  pythonImportsCheck = [
+    "mitm6"
+  ];
+
+  meta = {
+    description = "DHCPv6 network spoofing application";
+    homepage = "https://github.com/dirkjanm/mitm6";
+    changelog = "https://github.com/dirkjanm/mitm6/releases/tag/v${version}";
+    license = lib.licenses.gpl2Plus;
```

```suggestion
    license = lib.licenses.gpl2Only;
```
see https://github.com/dirkjanm/mitm6/blob/8e75884ab7c690d422b7da759c2f6d5426d8ef46/setup.py#L16